docs/sources/reference/commandline/cli.md

```diff
@@ -98,16 +102,16 @@ you can write `docker run -it --name test busybox sh`.
 
 ### Boolean
 
-Boolean options take the form `-d=false`. The value you see in the help text is the
-default value which is set if you do **not** specify that flag. If you specify
-a Boolean flag without a value, this will set the flag to `true`, irrespective
-of the default value.
+Boolean options take the form `-d=false`. The value you see in the help text is
+the default value which is set if you do **not** specify that flag. If you
+specify a Boolean flag without a value, this will set the flag to `true`,
+irrespective of the default value.
 
-For example, running `docker run -d` will set the value to `true`, so
-your container **will** run in "detached" mode, in the background.
+For example, running `docker run -d` will set the value to `true`, so your
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1347209/7846982/cbb7d23a-0475-11e5-9ab2-3ec22850fff3.png)

Also, don't need quotes around detached here and no. so --->

For example, the `docker run-d` command causes Docker to use `true` and your container then runs in the detached mode.